backend/pkg/abstraction/logger.go

```diff
@@ -16,6 +16,8 @@ type LoggerRequest interface {
 // Logger is the module in charge of storing and retrieving information from the
 // hard drive, making it persistent.
 type Logger interface {
+	Start() error
+	Stop() error
```

Comment:
notice you have added the `startKeys` and `stopKeys` to the Start and Stop methods on the `Logger`, you will have to change this (and also the `data.Logger`, which will be a bit weird since the start and stop keys does not apply to it, so just pass a nil slice to the `data.Logger` start and stop methods)